src/components/Answer.jsx

```diff
@@ -146,20 +146,28 @@ const Answer = (props) => {
       />
     ) : null;
 
+    const _renderQuestionCommentIcon = () => {
+      if (options.enableComments) {
+        return (
+            <Col className="no-padding-left" lg="auto">
+              <QuestionCommentIcon
+                  question={question}
+                  onChange={props.onCommentChange}
+              />
+            </Col>
+        );
+      } else return null;
+    }
+
     return (
         <Row>
           <Col className="no-padding-right" lg="auto">{label}</Col>
           {questionHelp ?
               <>
                 <Col className="no-padding-left" lg="1" >{questionHelp}</Col>
-                <Col className="no-padding-left" lg="auto"><QuestionCommentIcon question={question} onChange={props.onCommentChange}/></Col>
+                {_renderQuestionCommentIcon()}
```

Comment:
I consider this little bit strange as can add other types of icons ... as we have following things handle-ing here:
1) define order in which icons will be shown
2) layout icon in a nice way
3) for each icon show it/hide it based on some external function
 
It seems to me that it would be great to separate the concerns. E.g. "logic of 3)" is here on different level as "questionHelp ? ..." solves it directly here, while "this.context.options.enableComments" solves it inside of  "_renderQuestionComment" method. 

In detail :
1) should be responsibility  of generic question as it will be reused in WizzardStep or any extension of Question:
      =>   static method of the Question would fit here
2) should be responsibility of specific question component but the gaps between components should be i guess in most cases same => static method of the Question would fit here for default implementation, while each component could reuse it/override it and fit to its content
3) each component (questionHelp/comment/ ...) should evaluate itself if it should show or not (e.g. parent component can pass function to be evaluated)

Reply:
Just to summarize, we want to unify handling of questionHelp and questionComment as more icons are to come :)